code/modules/mob/living/status_procs.dm

```diff
@@ -0,0 +1,512 @@
+	var/new_value = directional_bounded_sum(eye_blind, amount, bound_lower, bound_upper)
+	SetEyeBlind(new_value)
+
+// EYE_BLURRY
+
+/mob/living/EyeBlurry(amount)
+	SetEyeBlurry(max(eye_blurry, amount))
+
+/mob/living/SetEyeBlurry(amount)
+	var/old_val = eye_blurry
+	eye_blurry = max(amount, 0)
+	// We transitioned to/from 0, so update the eye blur overlays
+	if((old_val == 0 || eye_blurry == 0) && (old_val != eye_blurry))
+		update_blurry_effects()
+
+/mob/living/AdjustEyeBlurry(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(eye_blurry, amount, bound_lower, bound_upper)
+	SetEyeBlurry(new_value)
+
+// HALLUCINATION
+
+/mob/living/Hallucinate(amount)
+	SetHallucinate(max(hallucination, amount))
+
+/mob/living/SetHallucinate(amount)
+	hallucination = max(amount, 0)
+
+/mob/living/AdjustHallucinate(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(hallucination, amount, bound_lower, bound_upper)
+	SetHallucinate(new_value)
+
+// JITTER
+
+/mob/living/Jitter(amount, force = 0)
+	SetJitter(max(jitteriness, amount), force)
+
+/mob/living/SetJitter(amount, force = 0)
+	// Jitter is also associated with stun
+	if(status_flags & CANSTUN || force)
+		jitteriness = max(amount, 0)
+
+/mob/living/AdjustJitter(amount, bound_lower = 0, bound_upper = INFINITY, force = 0)
+	var/new_value = directional_bounded_sum(jitteriness, amount, bound_lower, bound_upper)
+	SetJitter(new_value, force)
+
+// LOSE_BREATH
+
+/mob/living/LoseBreath(amount)
+	SetLoseBreath(max(losebreath, amount))
+
+/mob/living/SetLoseBreath(amount)
+	losebreath = max(amount, 0)
+
+/mob/living/AdjustLoseBreath(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(losebreath, amount, bound_lower, bound_upper)
+	SetLoseBreath(new_value)
+
+// PARALYSE
+
+/mob/living/Paralyse(amount, updating = 1, force = 0)
+	SetParalysis(max(paralysis, amount), updating, force)
+
+/mob/living/SetParalysis(amount, updating = 1, force = 0)
+	if(status_flags & CANPARALYSE || force)
+		paralysis = max(amount, 0)
+		if(updating)
+			update_canmove()
+			update_stat()
+
+/mob/living/AdjustParalysis(amount, bound_lower = 0, bound_upper = INFINITY, updating = 1, force = 0)
+	var/new_value = directional_bounded_sum(paralysis, amount, bound_lower, bound_upper)
+	SetParalysis(new_value, updating, force)
+
+// SILENT
+
+/mob/living/Silence(amount)
+	SetSilence(max(silent, amount))
+
+/mob/living/SetSilence(amount)
+	silent = max(amount, 0)
+
+/mob/living/AdjustSilence(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(silent, amount, bound_lower, bound_upper)
+	SetSilence(new_value)
+
+// SLEEPING
+
+/mob/living/Sleeping(amount, updating = 1)
+	SetSleeping(max(sleeping, amount), updating)
+
+/mob/living/SetSleeping(amount, updating = 1)
+	sleeping = max(amount, 0)
+	update_sleeping_effects()
+	if(updating)
+		update_stat()
```

Comment:
`update_stat` *would* handle falling unconscious/getting back up, but that's still handled in the other refactor. `update_canmove` should be called in both, for consistency's sake, though iirc `update_stat` calls `update_canmove` if the person falls unconscious or comes back awake.